mkdocs.yml

```diff
@@ -52,6 +52,7 @@ nav:
   - Home: index.md
   - Get Started: get-started.md
   - Downloads: downloads.md
+  - Updates: updates.md
```

Comment:
I like this idea and is close to what I envisioned :) But I think we should keep each update as a separate markdown file _under_ this section:

* Gives us an atom link to each update
* Removes the problem of an infinitely long page once we've done more updates

Reply:
Understand. New suggestion coming up...